packages/server/src/process/finish.rs

```diff
@@ -123,27 +121,6 @@ impl Server {
 			.try_collect::<Vec<_>>()
 			.await?;
 
-		// If any of the children were canceled, then this process should be canceled.
```

Comment:
Now that it's up to the parent process to handle cancellation beneath it, this code is no longer necessary.